src/pages/Register/index.tsx

```diff
@@ -39,12 +43,16 @@ const Register = () => {
       setMessage('Passwords do not match')
     }
 
-    if (role === jobseeker) {
-      dispatch(registerJobseekerRequest(email, password))
-      setMessage('Registered successfully')
-    } else if (role === employer) {
-      dispatch(registerEmployerRequest(email, password))
-      setMessage('Registered successfully')
+    if (password === confirmPassword) {
+      if (role === jobseeker) {
+        dispatch(registerJobseekerRequest(email, password))
+      }
+      if (role === employer) {
+        dispatch(registerEmployerRequest(email, password))
+      }
+      // setMessage('Registered successfully')
+      // setTimeout(() => history.push('/login'), 2000)
+      dispatch(redirectPage(history))
```

Comment:
No, you pass the `history` object to `registerEmployerRequest` and `registerJobseekerRequest`. Because only inside register saga you can check the register status